ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_network_services.schema.yml

```diff
@@ -79,6 +79,11 @@ $defs:
             By default the iBGP peering subnet is redistributed into the overlay routing protocol per VRF.
             Setting `redistribute_mlag_ibgp_peering_vrfs: false` under a tenant will change this default to prevent redistribution of these subnets for all VRFs in the tenant.
             This setting can be overridden per VRF.
+        evpn_vlan_bundle:
+          type: str
+          description: |-
+            Name of the "evpn_vlan_bundle" defined under "evpn_vlan_bundles".
+            This settings enables the "evpn_vlan_bundle" setting for all l2vlans and SVIs present in the tenant.
```

Comment:
Name of a bundle defined under 'evpn_vlan_bundles' to inherit configuration for all l2vlans and SVIs present in the tenant.

Reply:
None of them are really good IMO.

Reply:
Added better description.